app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
Why we have included scoped views ?

Reply:
I couldn't understand it, will you please explain it to me.